pkg/trace/obfuscate/sql_test.go

```diff
@@ -1074,3 +1189,253 @@ func TestUnicodeDigit(t *testing.T) {
 	o := NewObfuscator(nil)
 	o.ObfuscateSQLString(hangStr)
 }
+
+func TestMySQLDialect(t *testing.T) {
+	// These queries specifically test that obfuscation produces the same result on the raw query and MySQL digest query
+	rawQuery := "SELECT `avg_us`, `ro` as `percentile` FROM" +
+		"(SELECT `avg_us`, @rownum := @rownum + 1 as `ro` FROM" +
+		"    (SELECT ROUND(avg_timer_wait / 1000000) as `avg_us`" +
+		"        FROM performance_schema.events_statements_summary_by_digest" +
+		"        ORDER BY `avg_us` ASC) p," +
+		"    (SELECT @rownum := 0) r) q" +
+		"    WHERE q.`ro` > ROUND(.95*@rownum)" +
+		"    ORDER BY `percentile` ASC" +
+		"    LIMIT 1"
+	digestQuery := "SELECT `avg_us` , `ro` AS `percentile` FROM ( SELECT `avg_us` , @? := @? + ? AS `ro` FROM ( SELECT `ROUND` ( `avg_timer_wait` / ? ) AS `avg_us` FROM `performance_schema` . `events_statements_summary_by_digest` ORDER BY `avg_us` ASC ) `p` , ( SELECT @? := ? ) `r` ) `q` WHERE `q` . `ro` > `ROUND` ( ? * @? ) ORDER BY `percentile` ASC LIMIT ?"
+	expectedDenorm := "SELECT avg_us, ro FROM ( SELECT avg_us, @rownum := @rownum + ? FROM ( SELECT ROUND ( avg_timer_wait / ? ) FROM performance_schema.events_statements_summary_by_digest ORDER BY avg_us ASC ) p, ( SELECT @rownum := ? ) r ) q WHERE q. ro > ROUND ( ? * @rownum ) ORDER BY percentile ASC LIMIT ?"
+	expectedDenormDigest := "SELECT avg_us, ro FROM ( SELECT avg_us, @? := @? + ? FROM ( SELECT ROUND ( avg_timer_wait / ? ) FROM performance_schema . events_statements_summary_by_digest ORDER BY avg_us ASC ) p, ( SELECT @? := ? ) r ) q WHERE q . ro > ROUND ( ? * @? ) ORDER BY percentile ASC LIMIT ?"
+	expectedNormalized := "SELECT avg_us, ro FROM ( SELECT avg_us, @? := @? + ? FROM ( SELECT ROUND ( avg_timer_wait / ? ) FROM performance_schema.events_statements_summary_by_digest ORDER BY avg_us ASC ) p, ( SELECT @? := ? ) r ) q WHERE q.ro > ROUND ( ? * @? ) ORDER BY percentile ASC LIMIT ?"
+
+	cases := []sqlTestCaseNormalize{
+		{
+			rawQuery,
+			expectedDenorm,
+			expectedNormalized,
```

Comment:
Instead of creating these variables, can you inline them like all the other tests please?